styles.css

```diff
@@ -128,23 +175,1223 @@ ul {
   display: flex;
   width: 327px;
   height: 164px;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
   color: #172b4d;
   flex: none;
   order: 1;
   flex-grow: 0;
 }
 
 .social_icon_container {
-  display: flex;
-  justify-content: center;
+  width: 170.17px;
+  height: 20px;
+  flex: none;
   order: 2;
   flex-grow: 0;
 }
 
-.social-icon {
+.social_icon_container ul {
+  position: absolute;
+  width: 170.17px;
+  height: 20px;
+  justify-content: space-between;
+}
+
+.twitter {
+  /* position: absolute; */
+  left: 27.31%;
+  right: 67.36%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.github {
+  /* position: absolute; */
+  left: 59.04%;
+  right: 36.16%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.linkedin {
+  /* position: absolute; */
+  left: 37.71%;
+  right: 56.96%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+.medium {
+  /* position: absolute; */
+  left: 48.38%;
+  right: 46.29%;
+  top: 74.1%;
+  bottom: 23.69%;
+  background: #fff;
+}
+
+.angellist {
+  /* position: absolute; */
+  left: 68.91%;
+  right: 27.31%;
+  top: 73.83%;
+  bottom: 23.42%;
+  background: #fff;
+}
+
+/* .social-icon {
   flex-direction: column;
   fill: #505f79;
   padding-left: 1.6em;
   width: 2em;
 }
+.scroll {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 8px;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    left: calc(50% - 40px/2 + 0.5px);
+    bottom: 4px;
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+} */
+.action {
+  width: 327px;
+  height: 48px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.normal_button {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: calc (50% - 40px / 2 + 0.5px);
+  background: #fff;
+  border-radius: 32px;
+}
+
+.recent_works {
+  display: grid;
+  grid-template-columns: 379px;
+  grid-template-rows: 97px 4px repeat(6, 474px);
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 379px;
+  height: 3283px;
+  left: 0;
+  top: 708px;
+  justify-content: end;
+}
+
+.primary_txt_R {
+  grid-row: 1;
+
+  /* width: 327px;
+  height: 97px; */
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.indicator {
+  grid-row: 2;
```

Comment:
[OPTIONAL] There is a blank empty space on the right side of your page which is not present in the design as shown below. Kindly ensure that you remove it in order to make your website match the design. :pray: 

![Screenshot from 2022-12-03 23-45-06](https://user-images.githubusercontent.com/65233860/205467100-580a014d-e2d2-4742-9319-70b98da09673.png)